experiment/run_experiment.py

```diff
@@ -237,7 +237,7 @@ def set_up_fuzzer_config_files(fuzzer_configs):
 def start_experiment(experiment_name: str, config_filename: str,
                      benchmarks: List[str], fuzzer_configs: List[dict]):
     """Start a fuzzer benchmarking experiment."""
-    check_no_local_changes()
+    # check_no_local_changes()
```

Comment:
Debug code.
Although frankly I find this check very annoying. Esp for development. Maybe we should allow silencing it with an env var @inferno-chromium ?

Reply:
Please refresh. I decomment things and check format now.